autoreject/autoreject.py

```diff
@@ -42,28 +42,38 @@ def _slicemean(obj, this_slice, axis):
     return mean
 
 
-def validation_curve(epochs, y, param_name, param_range, cv=None):
+def validation_curve(epochs, y=None, param_name="thresh", param_range=None,
+                     cv=None, return_param_range=False):
     """Validation curve on epochs for global autoreject.
 
     Parameters
     ----------
     epochs : instance of mne.Epochs.
         The epochs.
-    y : array
+    y : array | None
         The labels.
     param_name : str
         Name of the parameter that will be varied.
-    param_range : array
+        Defaults to 'thresh'.
+    param_range : array | None
         The values of the parameter that will be evaluated.
+        If None, 15 values between the min and the max threshold
+        will be tested.
```

Comment:
The problem doing this is that you will have `np.inf` as the error when the candidate threshold is the min threshold. Even if you do something like min threshold + epsilon (so that at least one epoch is retained), you have the same problem as for some folds it could end up being the min threshold

Reply:
problem with None param or with a my custom grid?